tests/Integration/DataManagementSystem/TestClientFTS3.py

```diff
@@ -7,245 +7,246 @@
 from DIRAC.Core.Base.Script import parseCommandLine
 parseCommandLine()
 
-import unittest
```

Comment:
I do, but there's nothing called FTS3Client (anymore) so I don't what to do. I guess this your stuff now :)